srcpkgs/lightdm-slick-greeter/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'lightdm-slick-greeter'
+pkgname=lightdm-slick-greeter
+version=1.4.1
+revision=1
+wrksrc="slick-greeter-${version}"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --sbindir=/usr/bin --libexecdir=/usr/lib/lightdm"
```

Comment:
These should be redundant because build style `gnu-configure` defines them. Try without.